src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -524,4 +524,49 @@ trait ScalaSettings extends AbsScalaSettings
     else
       None
   }
+
+  object YsplainChoices extends MultiChoiceEnumeration {
+    val enable = Choice("enable", "activate the splain error formatting engine")
+    val noColor = Choice("no-color", "don't colorize type errors formatted by splain")
+    val noBoundsImplicit = Choice("no-bounds-implicit", "suppress any implicit bounds errors")
+    val noTree = Choice("no-tree", "don't display implicit chains in tree layout")
+    val verboseTree = Choice("verbose-tree", "display all intermediate implicits in a chain")
+    val noInfix = Choice("no-infix", "don't format infix types")
+    val noFoundReq = Choice("no-found-req", "don't format found/required type errors")
+  }
+
+  val Ysplain: MultiChoiceSetting[YsplainChoices.type] =
+    MultiChoiceSetting(
+      name = "-Ysplain",
```

Comment:
What do you think of a name like `-Yexplain-implicits` instead?

Reply:
I have no reservations about getting rid of the branding. whatever sounds good to the team.

Reply:
Actually `-Vimplicits` in 2.13. Maybe two options is not necessary. But it would be cool to keep `-Ysplain` as an alias.

Reply:
Let's pick one :) I think `-Yexplain-implicits` is better because users new to splain won't know it's about implicits at all if they see the option in a build.

Reply:
Som's right that it should be a `-V` something something, though, isn't he?

Reply:
`-V` is for "verbose", right? In that case, maybe `-Vimplicit-errors`?

Reply:
I'm confused now! Should it be `-Vexplain-implicits`?

Reply:
oops, didn't see that last comment

Reply:
Let's just pick one ;)

Reply:
vexplain is my new favourite word.

Reply:
I don't mind `-Vimplicits`. The simplified options avoid compounds that are hard to remember or discover. (I could see `-Vexplain` with `-Vexplain:types,implicits` and retire `-explaintypes`. But "verbose" is redundant with "explain".)

Reply:
So should we start with `-Vexplain` and integrate it with `-explaintypes` later?

Reply:
marking as resolved because this discussion is now stale (there is further discussion on this in the December round of feedback) and because the issue is on our March-2021 list of blockers